F3_PA-examplescripts.Altis/f/radios/tfr/fn_tfr_addRadios.sqf

```diff
@@ -34,65 +39,57 @@ switch ((side player)) do { //longrange, shortrange, rifradio
 // a set of radios.
 
 if(_typeOfUnit != "NIL") then {
+  // Set the list of units that get a rifleman's radio
```

Comment:
Note that the changes in this block are just dropping the "radios disabled" check and fixing the indentation.